### DISCUSSION TITLE
Chat Heading issue

### DISCUSSION THREAD

**Discussion by 1orZero** [olimorris/codecompanion.nvim#182] [Q&A] [answered]
Hi @olimorris , I just found this awesome plugin in Reddit! I like how this plugin already implemented so many Cursor (vscode fork editor) liked features. Especially the `/buffers` commands to easily insert any opened buffers in the chat! Its so useful!

But I have found that when I use a custom model with openai adapter, its has different behaviour. Is this normal?

Using a custom model name does not insert a new heading for the response.
https://github.com/user-attachments/assets/78fc0a20-2393-4911-91b0-bc3ca80eb0bb

Using `gpt-4o` inserted a heading before and after the response. 
https://github.com/user-attachments/assets/0934fc69-1ef7-47a6-b566-1e6b8353fbcd

**Comment by 1orZero** [comment]
OpenAi's official model name
<img width="1448" alt="SCR-20240909-byuv" src="https://github.com/user-attachments/assets/45753a22-f03f-45db-be0a-a798e3734087">

Custom model name
<img width="1651" alt="SCR-20240909-byzs" src="https://github.com/user-attachments/assets/acc5151e-cec9-483f-bfd6-0547a52248fc">

**Comment by olimorris** [comment]
Thanks for raising. I can't think what this could be given the endpoints are the same.

Can you raise a bug report and share the log file? Testing for just the gpt-4o-plus

**Comment by 1orZero** [comment, marked as answer]
I have just created an [issue](https://github.com/olimorris/codecompanion.nvim/issues/183) with the minimal config. Thanks again for making this great plugin!


Please let me know if you need a custom endpoint & a custom model to test this.
I can DM you the credentials
